src/swell/utilities/run_jedi_executables.py

```diff
@@ -112,14 +112,19 @@ def run_executable(
     np: int,
     jedi_executable_path: str,
     jedi_config_file: str,
-    output_log: str
+    output_log: str,
+    perhost: int=None
 ) -> None:
 
     # Run the JEDI executable
     # -----------------------
-    logger.info('Running '+jedi_executable_path+' with '+str(np)+' processors.')
 
-    command = ['mpirun', '-np', str(np), jedi_executable_path, jedi_config_file]
+    if perhost is None:
+       logger.info('Running '+jedi_executable_path+' with '+str(np)+' processors.')
+       command = ['mpirun', '-np', str(np), jedi_executable_path, jedi_config_file]
+    else:
+       logger.info('Running '+jedi_executable_path+' with '+str(np)+' processors & perhost '+str(perhost))
+       command = ['mpirun', '-np', str(np), '-perhost', str(perhost), jedi_executable_path, jedi_config_file]
```

Comment:
This fixes `pycodestyle` complaints about indentation and line length.

(Also, I'm personally partial to the f-string syntax -- `f"the value of x is {x}"` --- over string concatenation --- `"the value of x is "+x` --- but to each their own...).

```suggestion
    if perhost is None:
        logger.info(f"Running {jedi_executable_path} with {str(np)} processors.")
        command = [
            'mpirun',
            '-np', str(np),
            jedi_executable_path,
            jedi_config_file
        ]
    else:
        logger.info(
            f"Running {jedi_executable_path} with {str(np)} processors & perhost {str(perhost)}"
        )
        command = [
            'mpirun',
            '-np', str(np),
            '-perhost', str(perhost),
            jedi_executable_path,
            jedi_config_file
        ]
```

Reply:
@ashiklom can you explain to me what was the problem with/ the original set of lines here? What you have is identical to what I have except that the command is split into multiple lines!

Reply:
Our "python coding norms" automated test was failing because the lines were too long (>100 characters). So I split the lines so our checker approves this.

https://github.com/GEOS-ESM/swell/actions/runs/13970973305/job/39112388175#step:6:15